variants/vueapp/.eslintrc.cjs

```diff
@@ -0,0 +1,21 @@
+/* eslint-env node */
+require("@rushstack/eslint-patch/modern-module-resolution");
+
+module.exports = {
+  "root": true,
+  "extends": [
+    "plugin:vue/vue3-essential",
+    "eslint:recommended",
+    "@vue/eslint-config-prettier"
+  ],
+  "overrides": [
+    {
+      "files": [
+        "cypress/e2e/**.{cy,spec}.{js,ts,jsx,tsx}"
```

Comment:
Is this necessary? We don't use typescript, and `.vue` but not `.{j,t}sx`.

Reply:
I used ESLint for formatting